multinet/db.py

```diff
@@ -300,6 +325,26 @@ def workspace_table_keys(
     return keys
 
 
+# TODO: Ensure current user has writer role
```

Comment:
This should be on the API endpoint like the rest of them, not in the db file. It can likely be added with this PR if we wait until #412 lands or we can wait and add it after. If we wait, let's open an issue

Reply:
I think it makes the most sense to address this in a separate PR, after #412 lands. I'll create an issue for it.

Reply:
#416